packages/markers/src/browser/problem/problem-contribution.ts

```diff
@@ -99,4 +112,11 @@ export class ProblemContribution extends AbstractViewContribution<ProblemWidget>
             await model.selectNode(firstChild);
         }
     }
+
+    protected withWidget<T>(widget: Widget | undefined = this.tryGetWidget(), cb: (problems: ProblemWidget) => T): T | false {
```

Comment:
I guess the `problems` argument in the callback could also be `undefined` ? All 3 references (https://github.com/theia-ide/theia/pull/5071/files#diff-537b890524691310851c1b7e069dff27R82) of this function does not pass any widget with the call back.

also, i am not sure if this function could be given a better name - i am also bad with naming things. from my understanding, you meant `return something with the given widget`.